feat(aps): rework troubleshooting (#4338)
scaleway/docs-content: pages/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx

Run the following command:
```sh
nc -zv <server_ip> <vnc_port>
```
If the connection fails:
- Verify the correct VNC port in the **Overview** section of your Scaleway console.
- [Reboot the server](/apple-silicon/how-to/reboot-mac-mini/).
- Restart screen sharing via SSH (see below).

#### Verify the SSH server response
Run the command:
```sh
nc -zv <server_ip> 22
```
If the connection fails, [reboot the server](/apple-silicon/how-to/reboot-mac-mini/).

#### Check for connection blacklisting
If VNC connection attempts fail repeatedly, your connection may be blacklisted. [Rebooting](/apple-silicon/how-to/reboot-mac-mini/) clears the `fail2ban` table.

#### Verify your connection credentials
- VNC: Ensure correct username and password.
- SSH: Ensure your SSH key was provided before installation.

#### Verify remote access client compatibility
Use a compatible VNC client. See [this guide](/apple-silicon/how-to/access-remote-desktop-mac-mini/#comparison-matrix-of-remote-desktop-clients) for recommendations.

#### Reinstall the Mac mini
If all else fails, [reinstall macOS](/apple-silicon/how-to/reinstall-mac-mini/) via the Scaleway console. 
<Message type="important">

>   Reinstalling your Mac mini will delete all data on it's disk. Ensure you have a backup of your data before launching any reinstallation.

Reviewer: ```suggestion
  Reinstalling your Mac mini will delete all data on its disk. Ensure you have a backup of your data before launching any reinstallation.
```

</Message>

## Advanced configuration and security

### Enabling and configuring Packet Filter (pf)

> To restrict access to Screen Sharing:

Reviewer: ```suggestion
To restrict access to screen sharing:
```

```sh
sudo nano /etc/pf.conf
```
Add:
```sh
block in on en0 proto tcp from any to any port 5900
pass in on en0 proto tcp from <static IP> to any port 5900
```
Apply:
```sh
sudo pfctl -f /etc/pf.conf
```

### Restarting screen sharing via SSH
1. Connect via SSH:
```sh
ssh your_mac_mini_username@<your_mac_mini_ip>
```
2. Restart screen sharing:
```sh
sudo killall screensharingd
```

## Further troubleshooting

> If the issue persists, contact [Scaleway's support](https://console.scaleway.com/support) for further troubleshooting.

Reviewer 2: ```suggestion
If the issue persists, contact [Scaleway's support](https://console.scaleway.com/support) for assistance.
```
